Add Ryukaizen to Contributors list | Fixed Markdown, Typos & Bullets

ryukaizen opened the pull request.

ryukaizen committed:
Add Ryukaizen to Contributors list | Fixed Markdown, Typos & Bullets

ryukaizen committed:
Merge branch 'master' into master

Umesh-01 reviewed (commented):
@ryukaizen try to resolve the merge conflicts..

ryukaizen committed:
Merge branch 'master' into master

ryukaizen committed:
Markdown fixes from #45923

Resolved merge conflicts, though there were still a few edits that needed to be done, that is why opening this new PR.

ryukaizen commented:
@Umesh-01 roger!

Umesh-01 commented:
@ryukaizen you can refer to this [video](https://www.youtube.com/watch?v=JtIX3HJKwfo)

ryukaizen committed:
Merge branch 'master' into master

ryukaizen commented:
Got it, thanks @Umesh-01
